src/views/htmlcontent/src/css/color-theme.css

```diff
@@ -40,31 +40,55 @@
   }
 
   /* icons */
-  .icon.extendFullScreen {
+  .vscode-light .icon.extendFullScreen {
     /* ExtendToFullScreen_16x_vscode */
     background-image: url('data:image/svg+xml;charset=utf-8,%3Csvg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%2016%2016%22%3E%3Cstyle%3E.icon-canvas-transparent%7Bopacity:0;fill:#f6f6f6%7D.icon-vs-out%7Bfill:#f6f6f6%7D.icon-vs-bg%7Bfill:#424242%7D.icon-vs-fg%7Bfill:#f0eff1%7D%3C/style%3E%3Cpath%20class=%22icon-canvas-transparent%22%20d=%22M16%2016H0V0h16v16z%22%20id=%22canvas%22/%3E%3Cpath%20class=%22icon-vs-out%22%20d=%22M16%2016H0V0h16v16z%22%20id=%22outline%22%20style=%22display:%20none;%22/%3E%3Cpath%20class=%22icon-vs-fg%22%20d=%22M11%209.414l3-3V14H2V2h7.586l-3%203h2l-1%201H3v7h7V8.414l1-1v2zM4%2012h5V8H4v4z%22%20id=%22iconFg%22%20style=%22display:%20none;%22/%3E%3Cpath%20class=%22icon-vs-bg%22%20d=%22M7.586%206H3v7h7V8.414L7.586%206zM9%2012H4V8h5v4zM1%201v14h14V1H1zm13%2013H2V2h12v12zM9%206l2-2H9l1-1h3v3l-1%201V5l-2%202-1-1z%22%20id=%22iconBg%22/%3E%3C/svg%3E');
   }
 
-  .icon.exitFullScreen {
+  .vscode-dark .icon.extendFullScreen {
```

Comment:
do you know what happens in High-Contrast theme?

Reply:
added high contrast explicitly instead of falling back to light theme